org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -53,6 +128,46 @@
             contentType="org.eclipse.wildwebdeveloper.json"
             id="org.eclipse.wildwebdeveloper.json">
       </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.bowerrc"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.bower"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.jsconfig"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.tsconfig"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.package"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.omnisharp"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.typings"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.lerna"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.tslint"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.json.eslintrc"
+            id="org.eclipse.wildwebdeveloper.json">
+      </contentTypeMapping>
```

Comment:
As all those content-types are children of the already associated org.eclipse.wildwebdeveloper.json content-type, you shouldn't have to add new associations, the existing one from the base type should be enough.